### DISCUSSION TITLE
小白想学习js

### DISCUSSION THREAD

**Discussion by GalaxY778** [VickScarlet/lifeRestart#321] [Q&A]
小白自学前端  项目架构是咋整的嘞   俺按f12不知道怎么定位触发事件需要的js位置   求大神推荐网课或者书

**Comment by VickScarlet** [comment]
你可以从 `src/index.js` 这个入口文件一步步跟着往里看，其实也没几个文件，不懂的语法可以参考 [MDN](https://developer.mozilla.org/zh-CN/docs/Web/JavaScript)
